generators/server/templates/gradle.properties.ejs

```diff
@@ -66,7 +66,7 @@ jaxb_runtime_version=2.3.2
 
 # gradle plugin version
 micronaut_plugin_version=1.4.5
-git_properties_plugin_version=2.2.2
+git_properties_plugin_version=2.4.1
 <%_ if (!skipClient) { _%>
 gradle_node_plugin_version=2.2.4
```

Comment:
lets update to 3.4.0, which supports e.g. arm64 and has other improvements

Reply:
done.